examples/torque_driven_ocp/phase_transition_uneven_bars.py

```diff
@@ -0,0 +1,121 @@
+import biorbd_casadi as biorbd
```

Comment:
The example seems neat

Can you add a docstring as for the other examples? I am not sure I understand the name of the file. What bars are you referring too? Gymnastics? If so, I would prefer a name that describe what the file is designed to show (uneven_variable_number?)

You can add a test that call that file so it will automatically test your new feature :)